OsungTS-MC3/Views/FeedCollectionViewCell.swift

```diff
@@ -20,27 +20,106 @@ class FeedCollectionViewCell: UICollectionViewCell {
         return stackView
     }()
     
-    // 상단
+    // UpperBar
     private let feedUpperBar: UIStackView = {
```

Comment:
```suggestion
    private let feedUpperBarStackView: UIStackView = {
```

class가 StackView인건 바로 알아보면 읽기 편할거같아용